BUILD.bazel

```diff
@@ -0,0 +1,130 @@
+load(
+    "@gz//bazel/skylark:build_defs.bzl",
+    "GZ_FEATURES",
+    "GZ_ROOT",
+    "GZ_VISIBILITY",
+    "gz_configure_header",
+    "gz_export_header",
+    "gz_include_header",
+)
+
+package(
+    default_visibility = GZ_VISIBILITY,
```

Comment:
Consider adding 
`default_applicable_licenses = [GZ_ROOT + "physics:license"],` similar to https://github.com/gazebosim/gz-utils/blob/gz-utils2/BUILD.bazel

Reply:
You'd also add
```
license(
    name = "license",
    package_name = "gz-physics",
)
```